ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/main/java/de/adorsys/ledgers/baam/db/domain/ManagerAccess.java

```diff
@@ -0,0 +1,40 @@
+package de.adorsys.ledgers.baam.db.domain;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+
+import java.util.Set;
+import java.util.UUID;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ManagerAccess {
```

Comment:
```ManagerAccess```, like all Access entities, is a subclass of ```BankAccountAccess``` and so this here it should extend it.
This will save you from having to rewrite many fields already present in the said superclass, such as ```weight``` and ```id```.